target/ppc/insn32.decode

```diff
@@ -128,6 +128,10 @@
 %xx2_xb         1:1 11:5
 @XX2            ...... ..... ... uim:2 ..... ......... ..       &XX2 xt=%xx2_xt xb=%xx2_xb
 
+&XX3            xt xa xb
+%xx3_xa         2:1 16:5
```

Comment:
These fields aren't XX2/XX3 only, so we are calling them `%xx_x[tabc]` in other commits, e.g.: https://github.com/PPC64/qemu/pull/36/commits/60f567b9d2ad4714d5e61ee496a48c9e2a47a566